test/images/windows/Makefile

```diff
@@ -23,16 +23,24 @@ sub-build-%:
 	img_version=$(shell cat $*/VERSION); \
 	docker --tlsverify --tlscacert "$(DOCKER_CERT_PATH)/ca.pem" \
 		--tlscert "$(DOCKER_CERT_PATH)/cert.pem" --tlskey "$(DOCKER_CERT_PATH)/key.pem" \
-		-H "$(REMOTE_DOCKER_URL)" build --no-cache --pull -t "$(REGISTRY)/$*:$${img_version}" $*/
+		-H "$(REMOTE_DOCKER_URL)" build --no-cache --pull -t "$(REGISTRY)/$*:$${img_version}" \
+		-f $*/Dockerfile_windows $*/
 
 sub-push-%:
 	img_version=`cat $*/VERSION`; \
 	docker --tlsverify --tlscacert "$(DOCKER_CERT_PATH)/ca.pem" \
 		--tlscert "$(DOCKER_CERT_PATH)/cert.pem" --tlskey "$(DOCKER_CERT_PATH)/key.pem" \
 		-H "$(REMOTE_DOCKER_URL)" push "$(REGISTRY)/$*:$${img_version}"
 
+sub-repush-as-linux-%:
+	img_version=$(shell cat $*/VERSION); \
+	docker buildx build --progress=plain --no-cache --pull --output=type=registry --platform "linux/amd64" \
+		--build-arg SOURCE="$(REGISTRY)/$*:$${img_version}" -t "$(REGISTRY)/$*:$${img_version}"-linux-cache $*/
+
 all-build: $(foreach image, ${ALL_IMAGES}, sub-build-${image})
 
 all-push: all-build $(foreach image, ${ALL_IMAGES}, sub-push-${image})
 
-.PHONY: all-build all-push
+all-push-as-linux: all-push $(foreach image, ${ALL_IMAGES}, sub-repush-as-linux-${image})
```

Comment:
could we update the readme to include this new cache image: https://github.com/kubernetes/kubernetes/tree/master/test/images/windows#building-images

Reply:
Done